example16/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Vyloučeno comitovat target a vše co je v nem. Pokud nefunguje gitignore na zaklade ktereho by ti to nemelo nabizet soubory v targetu, tak opravit gitignore. 
Target se ti vytvori lokalne na zaklade buildu.